pandas/tests/io/parser/dtypes/test_dtypes_basic.py

```diff
@@ -466,3 +466,14 @@ def test_use_nullabla_dtypes_string(all_parsers, storage):
                 }
             )
         tm.assert_frame_equal(result, expected)
+
+
+def test_use_nullable_dtypes_ea_dtype_specified(all_parsers):
+    # GH#491496
+    data = """a,b
+1,2
+"""
+    parser = all_parsers
+    result = parser.read_csv(StringIO(data), dtype="Int64", use_nullable_dtypes=True)
```

Comment:
Do we have tests for `dtype="int64", use_nullable_dtypes=True`?

Reply:
test_use_nullabla_dtypes_and_dtype Same file